doc/source/release/1.11.0-notes.rst

```diff
@@ -31,45 +31,182 @@ Highlights of this release
 New features
 ************
 
+`scipy.integrate` improvements
+==============================
+- Added `scipy.integrate.qmc_quad` for quasi-Monte Carlo integration.
+- For an even number of points, `scipy.integrate.simpson` now calculates
+  a parabolic segment over the last three points which gives improved
+  accuracy over the previous implementation.
+
 `scipy.cluster` improvements
 ============================
+- ``disjoint_set`` has a new method ``subset_size`` for providing the size
+  of a particular subset.
 
 
 `scipy.interpolate` improvements
 ================================
 
+`scipy.constants` improvements
+================================
+- The ``quetta``, ``ronna``, ``ronto``, and ``quecto`` SI prefixes were added.
+
 
 `scipy.linalg` improvements
 ===========================
+- `scipy.linalg.det` is improved and now accepts nD-arrays.
+- `scipy.linalg.lu` is improved and now accepts nD-arrays. With the new
+  ``matrix_p`` switch the output permutation argument can be 1D ``(n,)``
```

Comment:
Apologies for not refixing this. After the reviews we re-renamed the keyword back to `p_indices`
```suggestion
  ``p_indices`` switch the output permutation argument can be 1D ``(n,)``
```